test/specs/prompts/editor.js

```diff
@@ -0,0 +1,45 @@
+var expect = require('chai').expect;
+var _ = require('lodash');
+var ReadlineStub = require('../../helpers/readline');
+var fixtures = require('../../helpers/fixtures');
+
+var Editor = require('../../../lib/prompts/editor');
+
+describe('`editor` prompt', function () {
+  beforeEach(function () {
+    this.previousVisual = process.env.VISUAL;
+    process.env.VISUAL = 'truncate --size 0';
```

Comment:
Any things we could do to get some input back to the prompt? It'd be nice if we could `expect(answer).to.equal('dummy answer');`.

The empty string is not super insightful in this case :(